src/reporter/format-utils.xsl

```diff
@@ -305,12 +347,13 @@
   </xsl:choose>
 </xsl:function>
 
-<!-- Compares $node with $node-to-compare-with and returns an HTML class accordingly: 'same' or 'diff'
+<!-- Compares $node with $node-to-compare-with and returns an HTML class accordingly: 'same', 'inner-diff' or 'diff'
   Set $expected to true if $node is in Expected Result. Set false if in Actual Result. -->
 <xsl:function name="test:comparison-html-class" as="xs:string">
   <xsl:param name="node" as="node()" />
   <xsl:param name="node-to-compare-with" as="node()?" />
   <xsl:param name="expected" as="xs:boolean" />
+  <xsl:param name="for-name" as="xs:boolean" />

```

Comment:
The name of the `for-name` parameter seemed a bit mysterious until I looked at how you were calling the function. Maybe a comment similar to the explanation of the `expected` parameter would be useful, such as, "Set $for-name to true only when comparing the names of elements, attributes, or processing instructions."

Reply:
Yes, it wasn't a good name. Updated to a more descriptive name and added a comment. ([change](https://github.com/xspec/xspec/pull/704/commits/292979b0c2f2b425c040655f45a83226571f51f7#diff-0a14ba5b65a347807a3dacfbf7ba4a93R352-R401))

Reply:
Thanks! I like the new name. (It helps that "focusing" is not a language keyword the way the word "for" is.)

I saw your latest commit and I think this PR is ready to be merged, unless you are waiting for other reviewers.